modules/accredited_representative_portal/app/controllers/accredited_representative_portal/v0/power_of_attorney_request_decisions_controller.rb

```diff
@@ -18,13 +18,14 @@ class PowerOfAttorneyRequestDecisionsController < ApplicationController
 
       def create
         reason = decision_params[:reason]
+        declination_reason = decision_params[:declination_reason] if params[declination_reason]
```

Comment:
```suggestion
        declination_reason = decision_params[:declination_reason] ? decision_params[:declination_reason] : nil
```
I think I told you the wrong thing here.  In your original code if `params[:declination_reason]` was false, the `declination_reason` wouldn't be defined and passing it to `mark_declined` later would have thrown an error.  This way it'll always be defined and won't break when passed in.